### PR TITLE
Update tlg5034.tlg001b & corrections to tlg5034.tlg001d

### PR DESCRIPTION
**Pull request by PRoryCallahan**
1.) Removed extraneous page number notes. 2.) Removed extra returns 3.) Made consistent paragraph tags to all scholia (a., b., etc.) 4.) Moved MS notations from OCR position to where Drachmann intended to be read in text 5.) few line breaks should have been paragraph and vice versa 6.) caught incorrect line #s 7.) added some elements in Drachmann that were missing in scan

### CODE REVIEW

**Review by lcerrato — APPROVED** [review]

File: data/tlg5034/tlg001d/tlg5034.tlg001d.perseus-grc1.xml [hunk 1]
```diff
@@ -82,9 +82,6 @@
     <revisionDesc>
       <change when="2022-02-01" who="Patrick Callahan">1.) Removed extraneous page number notes. 2.) Removed extra returns 3.) Made consistent paragraph tags to all scholia (a., b., etc.) 4.) Moved MS notations from OCR position to where Drachmann intended to be read in text 5.) few line breaks should have been paragraph and vice versa 6.) caught incorrect line #s 7.) added some elements in Drachmann that were missing in scan</change>
     </revisionDesc>
```
[the remark this answers is not in the document]
Comment (PRoryCallahan) [reply data/tlg5034/tlg001d/tlg5034.tlg001d.perseus-grc1.xml:84]: Yes, for some reason the Travis check failed with two change logs. Or at least that is how I read the failed test report. Still new at this so am ready to stand  corrected about how I should have approached the error.

[the remark this answers is not in the document]
Comment (lcerrato) [reply data/tlg5034/tlg001d/tlg5034.tlg001d.perseus-grc1.xml:84]: @PRoryCallahan I there were two `<revisionDesc>` added, which is incorrect. There can be multiple `<change>` tags, but all must be within one `<revisionDesc>`.

[the remark this answers is not in the document]
Comment (lcerrato) [reply data/tlg5034/tlg001d/tlg5034.tlg001d.perseus-grc1.xml:84]: For example:
```
<revisionDesc>
    <change when="2022-02-28" who="Lisa Cerrato">Added section level divs.</change>
    <change when="2022-01-11" who="Lisa Cerrato">Initial markup and encoding review.</change>
</revisionDesc>
```

**Comment by PRoryCallahan** [comment]
Thanks again, @lcerrato , for your patience and guidance! I re-wrote the <change> tag. One more book left to finish this little revision project!

**Review by lcerrato — APPROVED** [review]

**Comment by lcerrato** [comment]
@PRoryCallahan 
No, thank you!!

**Pull request merged**